config/hydroshare-config.yaml

```diff
@@ -1,57 +1,49 @@
 ### Local Configuration Variables ###
 HS_PATH: /home/hydro/hydroshare
-HOST_SSL_DIR: /home/hydro/hs-certs
-HS_DATABASE: pg.development.sql
+HOST_SSL_DIR: /home/hydro/hydroshare/nginx/cert-files
```

Comment:
I don't understand why there are changes in this file

Reply:
This file was not intended to be part of this PR. The changes shown are done separately by Hydroshare when rebuilt.
I added no new changes to this file.

Reply:
The changes were merged, we'll have to back them out then.